dockerGrader/test/p1/testMain.py

```diff
@@ -0,0 +1,45 @@
+# UW-Madison CS301 Test Script
+import argparse
+# import boto3
+from io import StringIO
+import json
+import os
+# import resource
+import subprocess
+import sys
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='CS301 test monitor')
+    parser.add_argument('-i', '--id', help='set student id', required=True)
+    parser.add_argument('-p', '--project', help='set project name', required=True)
+    args = parser.parse_args()
+
+    try:
+        os.remove("result.json")
+    except OSError:
+        pass
+
+    errorLog = {}
+    try:
+        subprocess.check_output(
```

Comment:
I don't think it makes sense to implement timeout inside the container.  Don't we want to kill the container from outside if it's taking too long?

Reply:
I would be incline to not have this wrapper and run test.py directly in the container (for simplicity) and implement any additional logic outside of the containers.